src/ocean/core/SmartUnion.d

```diff
@@ -254,6 +289,33 @@ private template MemberList ( uint i, size_t len, U )
     }
 }
 
+/*******************************************************************************
+
+    Evaluates to a ',' separated list of the names of the members of U, each
+    wrapped with quotes ("").
+
+    Template_Params:
+        i = U member start index
+        U = aggregate to iterate over
+
+    Evaluates to:
+        a ',' separated list of the names of the members of U, each wrapped with
+        quotes ("").
+
+*******************************************************************************/
+
+private template MemberStringList ( uint i, U )
+{
+    static if ( i == U.tupleof.length )
+    {
+        const MemberStringList = "";
+    }
+    else
+    {
+        const MemberStringList = `,"` ~ FieldName!(i, U) ~ `"` ~
+            MemberStringList!(i + 1, U);
+    }
+}
```

Comment:
Regarding both points (the location of the template and using a template vs CTFE), I simply wrote code in the same style as the existing code in this module. I agree that the CTFE solution looks a lot nicer, though, so will change that.